src/storage/src/source/postgres.rs

```diff
@@ -57,78 +57,80 @@ static WAL_LAG_GRACE_PERIOD: Duration = Duration::from_secs(5 * 60); // 5 minute
 static MAX_WAL_LAG: u64 = 100 * 1024 * 1024;
 
 trait ErrorExt {
-    fn is_recoverable(&self) -> bool;
+    fn is_definite(&self) -> bool;
 }
 
 impl ErrorExt for tokio::time::error::Elapsed {
-    fn is_recoverable(&self) -> bool {
-        true
+    fn is_definite(&self) -> bool {
+        false
     }
 }
```

Comment:
what code breaks if we remove this impl?

Reply:
A tokio timeout! https://github.com/MaterializeInc/materialize/blob/a9b3e50d95e37c1fe18973a94bfc2c7069b292a9/src/storage/src/source/postgres.rs#L673-L674